### DISCUSSION TITLE
I NEED TO GET RID OF LIGHTSPEED FILTER

### DISCUSSION THREAD

**Discussion by Laterorder** [3kh0/ext-remover#709] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
So basically my school uses lightspeed filter agent and it is like really hard to get rid of I need something to break the extension because it blocks everything that could be a proxy. It's the most annoying thing ever and I really need something that would remove the extension again it is an extension

**Comment by TheAnimemer** [comment, marked as answer]
you can use [this](https://jamal2.bypassly.co/) as a proxy. It's what I'm currently using, although you cannot access some websites with it, as certain websites block out proxy access on their own. Remember to also be aware that they have your wifi set to block certain websites as well. You can bypass the wifi blocker with just changing the DNS to 8.8.8.8 

Happy travels

**Reply by replays36DXD** [reply]
i don't have access to the DNS setting (or really any settings)

**Reply by TheAnimemer** [reply]
Lightspeed seems to have blocked the proxy somewhat recently as well. Shame. I don't have any info on changing the DNS with it blocked out in settings. For my school, we have two wifi connections. The main one, and a guest one. The IT department made it so that chromebooks cannot change network settings (like custom DNS) on the main one, but they seemingly forgot to block it on the guest wifi as well. If you have two available wifi connections, check and make sure both are blocked. If you have an unlimited phone plan you can connect your chromebook to your hotspot instead as well. Good luck with this one.
